src/constraints/electric_utility_constraints.jl

```diff
@@ -384,7 +384,7 @@ function add_elec_utility_expressions(m, p; _n="")
 
     if !isempty(p.s.electric_tariff.tou_demand_rates)
         m[Symbol("DemandTOUCharges"*_n)] = @expression(m, 
-            p.pwf_e * sum( p.s.electric_tariff.tou_demand_rates[r] * m[Symbol("dvPeakDemandTOU"*_n)][r, tier] 
```

Comment:
This explains why one tier still worked and multiple tiers broke - julia treated the single-value array as a scalar.